go.mod

```diff
@@ -1,9 +1,21 @@
 module github.com/thegreenwebfoundation/grid-intensity-exporter
 
-go 1.15
+go 1.17
 
 require (
 	github.com/cenkalti/backoff/v4 v4.1.0
 	github.com/prometheus/client_golang v1.8.0
-	github.com/thegreenwebfoundation/grid-intensity-go v0.0.0-20201120145608-ed4c07f2660c
+	github.com/thegreenwebfoundation/grid-intensity-go v0.1.1-0.20220622112548-6b75d66a9da8
```

Comment:
Needs https://github.com/thegreenwebfoundation/grid-intensity-go/pull/24 to be merged and released